### DISCUSSION TITLE
Code for IBM i Fridays: A Free Seiden Community Event - Seiden Group

### DISCUSSION THREAD

**Discussion by jwoehr** [codefori/vscode-ibmi#2159] [General]
Seiden Group hosts regular online gatherings for those who want to share, learn, solve problems, and contribute improvements to Code for i. [Fill out this form to join the announcement list](https://www.seidengroup.com/2024/05/13/code-for-ibm-i-fridays-a-free-seiden-community-event/)
![image](https://github.com/user-attachments/assets/b9807231-4a18-42c0-ad3f-64a441489192)

**Comment by worksofliam** [comment]
I typically attend and help answer questions, debug an issue or show off new features.
